validator/src/commands/run/execute.rs

```diff
@@ -269,6 +269,9 @@ pub fn execute(
         );
     }
     let vote_use_quic = value_t_or_exit!(matches, "vote_use_quic", bool);
+    if !vote_use_quic {
+        warn!("Submission of votes via UDP is deprecated.");
```

Comment:
not approving or declining the direction here, but can you deprecate the cli arg [the normal way](https://github.com/anza-xyz/agave/blob/864c45243b84d4a86561af35e340155f6be8224e/validator/src/cli.rs#L108)?

Reply:
Done in https://github.com/anza-xyz/agave/pull/8761/commits/0be9ee8e40da2a7422003ef7cca200e2c92636f9 , thanks for pointing this out!